rawpy/_rawpy.pyx

```diff
@@ -709,6 +710,19 @@ cdef class RawPy:
             self.ensure_unpack()
             return self.p.imgdata.rawdata.color.maximum
 
+    property camera_white_level_per_channel:
+        """
+        Per-channel saturation levels read from raw file metadata, if it exists. Otherwise None.
```

Comment:
`None` is never returned here.

Reply:
Hey @letmaik sorry for the late follow up on this, it's been a crazy past few months. I missed this. I've updated to return `None` if any of the `linear_max` values is zero. Please review when you can.